src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,77 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  todo: Todo
+  selectTodo: (id: number) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ selectTodo, todo }) => {
+  const [user, setUser] = useState<User>();
+
+  const fetchUser = async () => {
+    const userFromServer = await getUser(todo.userId);
```

Comment:
handle error

Reply:
oh I see , use try catch if you choose async await syntaxes,  follow one convention